run_tests.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
-from flake8_builtins import BuiltinsChecker
-
 import ast
 import unittest
 
+from flake8_builtins import BuiltinsChecker
+
```

Comment:
I'll do it, because it's your library after all, just please note that this will be breaking PEP8 in this section:

Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.